### DISCUSSION TITLE
MIT 6.858: Computer System Security - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#135] [Announcements]
# MIT 6.858: Computer System Security - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%B3%BB%E7%BB%9F%E5%AE%89%E5%85%A8/MIT6.858/

**Comment by Rrrruin** [comment]
panopto 需要学生账号吗

**Reply by ShaneHeX** [reply]
视频没权限看啊

**Reply by MedivhGO** [reply]
B站有人搬运了, 可以去B站上看.

**Comment by meixuan-stephanie** [comment]
可以分享一下你当时做的lab3 资料吗？ 谢谢！

**Reply by MedivhGO** [reply]
lab3的参考资料在实验介绍里有, 你也可以在github上搜搜, 有人原来也做过.

**Comment by buhuizuocaigou** [comment]
我想问下如果看完这个了，后续课程还有么关于网络安全的 谢谢啦

**Reply by dengyunsheng250** [reply]
CS161

**Reply by Jed12145** [reply]
ASU CSE466: Computer Systems Security：https://csdiy.wiki/%E7%B3%BB%E7%BB%9F%E5%AE%89%E5%85%A8/CSE466/
This is my blog about this course:https://www.yuque.com/lee2fish/wmds0i/qt4kteqy6gb10qkz

**Reply by zuesdevil** [reply]
可以看看国外的认证，攻防可以瞅瞅offsec的，OSCP在国内认可度也可以。但是现在涨价了，可以通过自己搜索能力找到他们的教材

**Comment by ShaneHeX** [comment]
有个系统安全的课程


获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Nir_enix ***@***.***>
发送时间: Monday, April 24, 2023 10:13:21 PM
收件人: PKUFlyingPig/cs-self-learning ***@***.***>
抄送: Xin He ***@***.***>; Comment ***@***.***>
主题: Re: [PKUFlyingPig/cs-self-learning] MIT 6.858: Computer System Security - CS自学指南 (Discussion #135)


我想问下如果看完这个了，后续课程还有么关于网络安全的 谢谢啦

―

**Reply by buhuizuocaigou** [reply]
请问有无课程网站呢

**Comment by Jed12145** [comment]
For the lab3,we can also learn the angr to understand the symbolic execution.

**Comment by pandacase** [comment]
22 的需要登陆才能看 video，可以把 url 的 2022 改成 2020，20 版的网址是可以直接进油管视频的

**Reply by mao-y-h** [reply]
22的点视频链接后的Youtube，应该就可以在Youtube上看吧

**Comment by Gassery** [comment]
没事试着玩的，没想到看到了最新的课程
2023课程：http://css.csail.mit.edu/6.858/2023/
2024课程：http://css.csail.mit.edu/6.858/2024/
25的没找到
2026课程：http://css.csail.mit.edu/6.566/2026/
虽然视频看不到，但课前阅读是有变化的，以及22年的VM下载不了，下载23年的VM就行了，至少作业能做了
